src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -202,11 +215,12 @@ export default class ContinuationInBusinessBc extends Mixins(CommonMixin) {
   @Watch('businessContactFormValid', { immediate: true })
   @Watch('getFolioNumberValid', { immediate: true })
   private onContinuationInBusinessBcValid (): void {
-    this.setContinuationInBusinessBcValid(
-      this.addressFormValid &&
-      this.businessContactFormValid &&
-      (!this.isPremiumAccount || this.getFolioNumberValid)
-    )
+    const allValid = this.addressFormValid &&
+                   this.businessContactFormValid &&
+                   (!this.isPremiumAccount || this.getFolioNumberValid)
+    this.setContinuationInBusinessBcValid(allValid)
+    // Update the overall Define Company Step validity
+    this.setDefineCompanyStepValidity(allValid)
```

Comment:
This looks like the correct solution, but I wonder if we still need `setContinuationInBusinessBcValid()` or whether it can be deleted. The value it sets (getContinuationIn.businessBcValid) is only used for the stepper.

I think you should try to combine this into setDefineCompanyStepValidity().

Reply:
Also, this change broke some unit tests. I would clean this up and then fix the unit tests if they're still broken.

Reply:
Hi Severin, 
- I just deleted the setContinuationInBusinessBcValid(), and updated the variables in Stepper (only use isDefineCompanyValid() ).
- I also added the method to validate NameTranslation Component in step 2